### ISSUE TITLE
Whats the default cache path?

### ISSUE THREAD

**Issue by paulcripps** [meatpaste/imgsizer#1]
Might seem like a daft question but I'm getting errors:

```
Calling Tag: {exp:imgsizer:size src='/images/uploads/foldername/test.png' width='122'}
ImageSizer Error: Unable to create placeholder cache folder
```

I've tried adding a folder called 'sized' (777) to:
- /images/
- /images/uploads/
- /images/uploads/foldername

And still have errors

**Comment by paulcripps** [comment]
note, adding 

`cache_path="/local-location/websitename/images/sized"`

Fixes it... however I'd prefer to stick with the default path and NOT rely on cache_path="" as we've got hundreds of images to size over three staging servers.

**Comment by paulcripps** [comment]
Also, it looks like the plugin is stripping transparency from PNGs, (they now have a black background). Is there a way to prevent this?

**Comment by meatpaste** [comment]
not a silly question, I will have a look and post a fix. I use the config variable in the EE3 site its running on so this part wasn't tested. Also going to move your PNG issue to a seperate thread.

**Comment by paulcripps** [comment]
Brilliant, thanks for the response, look forward to a fix.

In the mean time could you give a rough example of the config variable in the EE3 for future reference and users?

**Comment by meatpaste** [comment]
$config['imgsizer_cache_path'] = '/var/www/vhosts/websitefolder/httpdocs/cdn/';
$config['imgsizer_cache_url'] = '/cdn/';

**Comment by meatpaste** [comment]
Fixed in version 4.0.1
